app/utils/seda_signer.go

```diff
@@ -124,7 +131,10 @@ func (s *sedaKeys) IsLoaded() bool {
 
 // Reload reloads the signer from the key file.
 func (s *sedaKeys) reload() error {
-	keys, err := loadSEDAKeys(s.keyPath)
+	// Reload should run from the same process as the one that loaded the signer,
+	// so so the check for the encryption key should already have passed if we're
```

Comment:
double so’s

Reply:
That was just indicating the quality of my work 😅